### PR TITLE
Updated instructions in Readme on how to disable Expiry/CVC for custom cards

### PR DESCRIPTION
**Pull request by JamesJSY**
@DannyVancura @ThibaultKlein @ppatriotis

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,29 +170,42 @@ You can create your own card types and add or remove them to or from card number
...
     // Note: The image that will be displayed in the card number text field's image view when this card type has been detected will load from an asset with the name `cardType.name`.
-	
-	// If the Issuer Identification Number (the first six digits of the entered card number) of a card number 
-	// starts with anything from 1000 to 1111, the card is identified as being of type "MyCardType":
+
+    // If the Issuer Identification Number (the first six digits of the entered card number) of a card number 
+    // starts with anything from 1000 to 1111, the card is identified as being of type "MyCardType":
     public let identifyingDigits = Set(1000...1111)
-	
-	// MARK: - Optional
```
Comment (ppatriotis) [inline README.md]: is this not optional anymore?

File: README.md [hunk 2]
```diff
@@ -170,29 +170,42 @@ You can create your own card types and add or remove them to or from card number
...
+
+    // Not specifying this will load a default cvc image:
     public let cvcImage: UIImage? = UIImage(named: "MyCardTypeCVCImage")
-	
-	// Not specifying this will result in a 16 digit number, separated into 4 groups of 4 digits.
-	// The grouping of your card number type. The following results in a card number format
-	// like "100 - 0000 - 00000 - 000000":
-	public let numberGrouping = [3, 4, 5, 6]
+
+    // Not specifying this will result in a 16 digit number, separated into 4 groups of 4 digits.
```
Comment (ppatriotis) [inline README.md]: Default should be after the description

File: README.md [hunk 3]
```diff
@@ -170,29 +170,42 @@ You can create your own card types and add or remove them to or from card number
...
-	public let CVCLength = 4
-	
-	// Not specifying this will load a default cvc image:
+
+    // Not specifying this will assert a three digits long cvc:
+    public let CVCLength = 4
+
+    // MARK: - Optional
+
+    // Not specifying this will load a default cvc image:
```
Comment (ppatriotis) [inline README.md]: Please add description before default explanation

File: README.md [hunk 4]
```diff
@@ -170,29 +170,42 @@ You can create your own card types and add or remove them to or from card number
...
     public let identifyingDigits = Set(1000...1111)
-	
-	// MARK: - Optional
-	
-	// Not specifying this will assert a three digits long cvc:
-	public let CVCLength = 4
-	
-	// Not specifying this will load a default cvc image:
+
+    // Not specifying this will assert a three digits long cvc:
```
Comment (ppatriotis) [inline README.md]: Please add description before default explanation

File: README.md (hunk 1 above)
Comment (JamesJSY) [inline README.md]: When I created a custom card, the CVCLength is required. I think it is an inconsistency in the document so I changed that.

File: README.md (hunk 2 above)
Comment (JamesJSY) [inline README.md]: OK I will update that

Comment (JamesJSY) [inline README.md]: Updated

**Pull request merged**